yarn-project/merkle-tree/src/snapshots/append_only_snapshot.test.ts

```diff
@@ -24,7 +24,7 @@ describe('AppendOnlySnapshot', () => {
     () => tree,
     () => snapshotBuilder,
     async tree => {
-      const newLeaves = Array.from({ length: 2 }).map(() => randomBytes(32));
+      const newLeaves = Array.from({ length: 2 }).map(() => Fr.random().toBuffer());
```

Comment:
After having the modulus check in Barretenberg FR I had to update this ^. It's a bit suspicious that it didn't cause some issues before.